homeassistant/components/google_assistant_sdk/notify.py

```diff
@@ -18,7 +18,7 @@
     "de": ("Nachricht an alle {0}", "Nachricht an alle an {1} {0}"),
     "es": ("Anuncia {0}", "Anuncia en {1} {0}"),
     "fr": ("Diffuse {0}", "Diffuse dans {1} {0}"),
-    "it": ("Trasmetti {0}", "Trasmetti in {1} {0}"),
+    "it": ("Trasmetti a tutti {0}", "Trasmetti in {1} {0}"),
```

Comment:
https://support.google.com/assistant/answer/9071582?hl=it has:

- "Trasmetti [messaggio]".
- "Grida [messaggio]".
- "Dì a tutti [messaggio]".
- "Comunica che [messaggio]".

Your variation isn't listed there. Can you confirm it works?

Do you also need to adjust the second command here that takes area/target?

Reply:
If what you have works it's fine. Unfortunately broadcast to a target is a hit or miss for non English languages. In the documentation I have: "Broadcast to specific rooms often doesn’t work for non-English languages."

Reply:
Hello, I tried this too and it works on my end.
I also think that `dì a tutti` should work, as it's the command that I usually use when I talk to the assistant by voice